package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve --proxy-config proxy.conf.json",
-    "build": "ng build",
+    "build": "ng build --prod",
     "test": "ng test",
     "test-ci": "ng test --code-coverage --karmaConfig=build/karma-ci.conf.js ",
-    "lint": "ng lint ndb-core --type-check",
+    "lint": "ng lint ndb-core --files src/app --type-check",
```

Comment:
any particular reason why we should stop checking files in other directories, like the main.ts or environments?